x-pack/plugins/maps/public/classes/layers/new_vector_layer_wizard/config.tsx

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+import React from 'react';
+import { LayerWizard, RenderWizardArguments } from '../../layers/layer_wizard_registry';
+import { NewVectorLayerEditor } from './index';
+import { DrawLayerIcon } from '../../layers/icons/draw_layer_icon';
+import { CREATE_DRAWN_FEATURES_INDEX_STEP_ID, ADD_DRAWN_FEATURES_TO_INDEX_STEP_ID } from './wizard';
+import { getFileUpload } from '../../../kibana_services';
+
+export const newVectorLayerWizardConfig: LayerWizard = {
+  categories: [],
+  description: i18n.translate('xpack.maps.newVectorLayerWizard.description', {
+    defaultMessage: 'Draw points & shapes and save in Elasticsearch',
```

Comment:
How about changing to "Draw points, lines, and polygons to Elasticsearch" so it has more symmetry with "Documents" layer description, "Points, lines, and polygons from Elasticsearch"? I think this will help connect the dots that the next time you add this layer to your map, you will just use the "Documents" layer.